pom.xml

```diff
@@ -29,6 +29,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.build.outputTimestamp>2024-02-11T10:49:01Z</project.build.outputTimestamp>
```

Comment:
likely same than https://github.com/yupiik/fusion/pull/5/files#r1485611027 outcome (flagging for tracing but not repeating the comment)